python/ray/data/_internal/datasource/parquet_datasource.py

```diff
@@ -654,7 +654,7 @@ def sample_fragments(
 
     sample_fragment = cached_remote_fn(_sample_fragment)
     futures = []
-    scheduling = local_scheduling or "SPREAD"
+    scheduling = local_scheduling or DataContext.get_current().scheduling_strategy
```

Comment:
Out-of-scope for this PR because we already do this elsewhere in `ParquetDatasource`, but we should use a sealed `DataContext` rather than calling `get_current()`. See https://github.com/ray-project/ray/pull/49096